dist/images/ovnkube.sh

```diff
@@ -1046,10 +1075,13 @@ ovn-node() {
     ${multicast_enabled_flag} \
     ${egressip_enabled_flag} \
     --ovn-metrics-bind-address ${ovn_metrics_bind_address} \
-    --metrics-bind-address ${ovnkube_node_metrics_bind_address} &
+    --metrics-bind-address ${ovnkube_node_metrics_bind_address} \
+     ${ovnkube_node_mode_flag} &
 
```

Comment:
Shall we remove these options in smart-nic-host mode? or maybe add conditional check when generating these options above (line 1035).

ovn_node_ssl_opts
metrics-bind-address

Reply:
> ovn_node_ssl_opts

yes we should not expose them for ovnkube-node in smart-nic-host mode.

> metrics-bind-address

So we are technically still starting an ovnkube node. we dont need to report metrics on it ?

https://github.com/ovn-org/ovn-kubernetes/blob/7f932cac4e69232c85483e06cd6b9772914fe04d/go-controller/cmd/ovnkube/ovnkube.go#L285

i think we would want metrics on ovnkube Node in all modes.